index.html

```diff
@@ -9,21 +9,57 @@
 </head>
 <body>
     
-    <h1>Promesse Seuglong Fongba</h1>
+   
     <div id="wrapper">
-<h2>About Me</h2>
-<img src="profile.png" height="300" />
-<p>I have always found inspiration in the quote, "life is like a roller coaster; it has its ups and downs, but it's your choice to scream or enjoy the ride." My life's journey has indeed resembled a roller coaster, shaped by my background and socioeconomic status. I was born in Cameroon, Africa, in a family with nine kids, and I'm the seventh. Now, I'm 30 years old, a mom to two kids (four and one-year-old), and I've been married since November 2017.</p>
-<p>Even though my family didn't have a lot of money when I was growing up, I never gave up on my dream of becoming a software engineer because I love working with technology. I recently moved to the United States, embarking on a new chapter of exploration and learning.</p>
-<p>Family is super important to me, and I cherish every moment I spend with them. I also really enjoy being outside, whether it's taking a walk in the park or having a picnic on the weekends. So, as I start this new phase of my life, I'm looking forward to the new experiences and opportunities it brings. Beyond academics, I have a flair for cooking. I find it extremely satisfying to experiment with new recipes and techniques, often inspired by the cooking shows I love to watch. The art of preparing delicious meals truly captivates me.</p>
-<h2>Expereience</h2>
+        <nav class="navbar">
+            <a href="#">About Me</a>
+            <a href="#">Experience</a>
+            <a href="#">Skills</a>
+            <a href="#">Projects</a>
+            <a href="#">Connect</a>
+            
+        </nav>
+        
```

Comment:
each of the hrefs should reference an ID so a user can click on the links and navigate to the section. This code would look something like this: ```html
        <nav class="navbar">
            <a href="#about">About Me</a>
            <a href="#experience">Experience</a>
            <a href="#skills">Skills</a>
            <a href="#projects">Projects</a>
            <a href="#connect">Connect</a>
        </nav>

Reply:
You should then add these ids to the sections you plan on navigating to. See later comments